src/main/java/juja/microservices/users/service/UserService.java

```diff
@@ -65,15 +86,63 @@ public UserService(UserRepository userRepository) {
                 .collect(Collectors.toList());
         logger.debug("Received response from repository: {}", users.toString());
 
-        List<UserDTO> result = users.stream()
-                .map(this::convertUserToUserDto)
-                .collect(Collectors.toList());
+        List<UserDTO> result = getConvertedResult(users);
         logger.debug("All users converted: {}", result.toString());
 
         return result;
     }
 
-    private UserDTO convertUserToUserDto(User user) {
-        return new UserDTO(user.getUuid(), user.getSlack(), user.getSkype(), user.getFullName());
+    @Scheduled(cron = "${cron.expression}")
+    public void scheduleUpdateUsers() {
+        updateUsersFromCRM();
+    }
+
+    public List<UserDTO> updateUsersFromCRM() {
+        Long lastUpdate = repository.findMaxLastUpdate();
+        if (lastUpdate == null) lastUpdate = 0L;
```

Comment:
I don't like such approach to write (in one row), and this is not on code convention.

Reply:
Done! I extracted it to the separate method